scalactic-test/src/test/scala/org/scalactic/DirectRequirementsSpec.scala

```diff
@@ -4783,7 +4783,7 @@ class DirectRequirementsSpec extends FunSpec with OptionValues {
 
     it("should throw NullArgumentException with correct message when one of passed parameters through object property is null") {
       class AClass {
-        val aNull: String = null
+        val aNull: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.